core/src/test/java/org/infinispan/statetransfer/RemoteGetDuringStateTransferTest.java

```diff
@@ -54,40 +63,88 @@
    /*
    Summary
 
-   T = initial topology, T1  = rebalance started but we have no data yet, T1.5 rebalance started and we have data, T2  = rebalance finished
-
-   The originator will retry the remote get if node 1 processes the request in T2.
-   If node 1 processes the request in T0 or T1, it still has the entry, and the originator doesn't have to retry.
+   T0   initial topology
+   T1   state transfer started
+   T2   state transfer finished but rebalance not complete
+   T3   read new, write all topology
+   T4   rebalance completed
```

Comment:
I had a few comments on the old PR here that you didn't handle, e.g. https://github.com/infinispan/infinispan/pull/4827#discussion-diff-101270697R70